config/.codegen.yaml

```diff
@@ -1,8 +1,12 @@
 schemapatch:
   requiredFeatureSets:
-  - ""
-  - "Default"
-  - "TechPreviewNoUpgrade"
-  - "CustomNoUpgrade"
+    - ""
+    - "Default"
+    - "TechPreviewNoUpgrade"
+    - "CustomNoUpgrade"
+  mustHaveOneOfClusterProfile:
```

Comment:
This would cause a logical tuple explosion as we run generation for every featureset for every clusterprofile.